### DISCUSSION TITLE
Does this mod disable achievements?

### DISCUSSION THREAD

**Discussion by dynital** [Rycieos/factorio-colorblind-ultimate#10] [Q&A] [answered]
I have a friend who's colorblind that has been struggling with the default accessibility settings.
I've suggested using a mod to cope with it and after some research I found this mod, however he holds his achievements in high regard.
I wanted to ask someone who has played the game with this mod if their achievements got disabled.
If you've read this far, thank you for your time.

**Comment by Rycieos** [comment, marked as answer]
I'll be honest, I have never actually tested it. I can test tomorrow. But as far as I know, Factorio always disables Steam achievements when any mod is active, even mods that don't modify gameplay, as it does not differentiate. This is greatly unfortunate, as many people, including myself, will forgo accessibility mods to get these achievements. Just another reason Wube should handle accessibility themselves instead of telling modders to handle it.

Edit: I have tested this, and can confirm [what the wiki contains](https://wiki.factorio.com/Achievements), which is:
* Factorio tracks achievements separately for non-modded and modded play.
* Only achievements earned during non-modded play will be granted as Steam achievements.
* In multiplayer, achievements can only be earned if the player has spent more than 50% of the time online. (I did not personally test this one.)
* Using any console command disables achievements completely for that save.

So the answer to your question is: not in game achievements, but Steam achievements yes. However, there are ways around this:
* Use a tool [like this one](https://forums.factorio.com/viewtopic.php?t=100924) to modify Factorio to report modded achievements to Steam.
* If your save used this mod previously and you want to restore the ability to earn Steam achievements, it is possible to [modify the save file to do so](https://forums.factorio.com/58077).
* Simply don't use this mod.
